README.md

```
@@ -8,17 +8,47 @@ Currently supports Windows - with macOS and Linux support planned. Setup scripts
 ## Getting Started
-Once you've cloned, run `scripts/Setup.bat` to generate Visual Studio 2022 solution/project files. Once you've opened the solution, you can run the WalnutApp project to see a basic example (code in `WalnutApp.cpp`). I recommend modifying that WalnutApp project to create your own application, as everything should be setup and ready to go.
+
+In order to clone the repository, you'll need to grab the submodules. To do that, you will need the following command line:
+
+``` bash
+git clone --recurse-submodules https://github.com/StudioCherno/Walnut.git
+```
+
+Once you've cloned the repository, you can run one of the following setup scripts to generate Visual Studio 2022 solution/project files. 
+
+*on Windows command line:*
```

markdownlint-cli2 (0.18.1)

Emphasis style
Expected: underscore; Actual: asterisk

(MD049, emphasis-style)